examples/Hello Wall/hello-wall.ifcx

```diff
@@ -1,9 +1,17 @@
 {
   "header": {
+    "id": "ifc5.technical.buildingsmart.org/examples/Hello Wall/hello-wall@v1.ifcx",
```

Comment:
is there any particular requirement for this ID? Could it also be "PROJ-ABC-STR-XYZ-0001@0012"?